### ISSUE TITLE
Implement Email Verification

### ISSUE THREAD

**Issue by jawooson** [gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities#26]
### Front End

Once a user registers an account, an email should be sent to the provided email address. That email should contain a verification link that the user will need to click on in order to confirm their account registration. After they click the verification link, ShowUp should show some sort of "Verification Complete!" page and either a link to the login page or they can already be logged in (your choice).

### Back End

Basically make it do what was described above. You will need a `verified` column in the database (T/F). Will probably also need a `logged_in` column (T/F).
